### PR TITLE
LPS-75355 Build lang

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by matuzalemsteles** [comment]
LGTM!

**Comment by diegonvs** [comment]
LGTM!

**Comment by interaminense** [comment]
LGTM!

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
@leoadb , the frontend team has already reviewed this code. This is good to go!

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_confirmation_dialog.js [hunk 1]
```diff
@@ -51,10 +55,12 @@ AUI.add(
 									}
 								]
 							},
-							width: false
+							width: config.width
 						},
+
```
Comment (leoadb) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_confirmation_dialog.js]: @alinedoleron Please remove this space. Not needed. Thanks!

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_confirmation_dialog.js [hunk 2]
```diff
@@ -51,10 +55,12 @@ AUI.add(
...
 						id: config.id,
 						title: config.title
+
```
Comment (leoadb) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_confirmation_dialog.js]: @alinedoleron Please remove this space. Not needed. Thanks!

File: dynamic-data-mapping-form-builder/src/main/resources/content/Language_en.properties [hunk 3]
```diff
@@ -23,7 +23,7 @@ data-provider-parameter-output=Data Provider's Output:
 data-provider-parameter-output-description=Assign the data provider's parameters you want to a field and the information will autofill it automatically.
 define-condition-and-action-to-change-fields-and-elements-on-the-form=Define condition(s) and action(s) to change fields and elements on the form.
 delete-current-page=Delete Current Page
-delete-field-question=Delete field?
+delete-field-question=Delete Field?
```
Comment (leoadb) [inline dynamic-data-mapping-form-builder/src/main/resources/content/Language_en.properties]: @pedroqueiroz94 We should do this in a separate commit. Is this change really necessary?

Comment (pedroqueiroz94) [reply dynamic-data-mapping-form-builder/src/main/resources/content/Language_en.properties]: Suggestion: Change delete-field-question to delete-field-window-title

**Comment by pedroqueiroz94** [comment]
Resent at https://github.com/liferay/com-liferay-dynamic-data-mapping/pull/617
